modules/features2d/misc/java/test/BRIEFDescriptorExtractorTest.java

```diff
@@ -69,12 +68,13 @@ public void testDescriptorType() {
     }
 
     public void testEmpty() {
-        assertFalse(extractor.empty());
+//        assertFalse(extractor.empty());
+        fail("Not yet implemented"); // BRIEF does not override empty() method
     }
 
     public void testRead() {
         String filename = OpenCVTestRunner.getTempFileName("yml");
-        writeFile(filename, "%YAML:1.0\ndescriptorSize: 64\n");
+        writeFile(filename, "%YAML 1.0\n---\ndescriptorSize: 64\n");
```

Comment:
By the way, dashes are needed based on YAML spec:
`A YAML stream may contain several independent YAML documents. A document header line may be used to start a document and must be used to separate documents within a stream. This line must start with a document separator: “---” followed by a line break or a sequence of space characters. If no explicit header line is specified at the start of the stream, the processor should behave as if a header line containing an unadorned “---” was specified. `